src/libraries/System.IO.FileSystem/tests/File/Move.netcoreapp.cs

```diff
@@ -0,0 +1,76 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Xunit;
+
+namespace System.IO.Tests
+{
+    public partial class File_Move : FileSystemTest
```

Comment:
I'm assuming all of the code that was moved between files wasn't modified at all and thus I don't need to review it, correct?

Reply:
Correct, no change aside from the "partial" keyword.